### PR TITLE
[JENKINS-37611] Add 'xunit' symbol

### PR DESCRIPTION
**Pull request by Dohbedoh**
@reviewbybees

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>1.596.1</version>
+        <version>1.642.3</version>
```
Comment (oleg-nenashev) [inline pom.xml]: IMHO it should be bumped to Parent POM 2.x instead. At least 2.something without upper bounds dependency checks. Would be helpful for other stories like JEP-200

**Comment by oleg-nenashev** [comment]
I would vote for #49 from @abayer

**Comment by Dohbedoh** [comment]
Thanks @oleg-nenashev. This can be probably be closed then.

**Comment by abayer** [comment]
Yeah, closing. =)

**Pull request closed without merging**